communication/moderation.md

```diff
@@ -42,6 +42,22 @@ These guidelines are for tool-specific policies that don't fit under a general u
 
 ### Mailing Lists
 
+### Moderating a SIG/WG list
+
+- SIG and Working Group mailing list should have parispittman@google.com and jorge@heptio.com as a coowner so that administrative functions can be managed centrally across the project.
+  - Moderation of the SIG/WG lists is up to that individual SIG/WG, these admins are there to help facilitate leadership changes, reset lost passwords, etc. 
+
+- Users who are violating the Code of Conduct or other negative activities (like spamming) should be moderated.
+  - [Lock the thread immediately](https://support.google.com/groups/answer/2466386?hl=en#) so that people cannot reply to the thread.
+  - [Delete the post](https://support.google.com/groups/answer/1046523?hl=en) - 
+  - In some cases you might need to ban a user from the group, follow [these instructions](https://support.google.com/groups/answer/2646833?hl=en&ref_topic=2458761#) on how stop a member from being able to post to the group. 
+
+For more technical help on how to use Google Groups, check the [Groups Help](https://support.google.com/groups/answer/2466386?hl=en&ref_topic=2458761) page.
+
+### New users posting to kubernetes-dev 
```

Comment:
We should encourage all SIGs use this. Many are complaining about spam and are locking down the group (via needing membership to read/post) rather than using this queue.